src/StdCheats.sol

```diff
@@ -692,6 +693,7 @@ abstract contract StdCheats is StdCheatsSafe {
     }
 
     function changePrank(address msgSender) internal virtual {
+        console2.log("changePrank is deprecated. Please use vm.startPrank instead.");
```

Comment:
Instead of importing console2 do you mind doing something this? It helps reduce compilation times in some cases, and compilation times are becoming a bottleneck for a lot of users :)  https://github.com/foundry-rs/forge-std/blob/c70ef65c9fe1c6255a83f6363f4c766fb67ba844/src/StdUtils.sol#L187-L197

Reply:
for sure! done in https://github.com/foundry-rs/forge-std/pull/434/commits/e6fae92e0adcc4c0308b90be9dba72b5b61db369